project_config.yml

```
@@ -4,9 +4,13 @@ schema_name: video_games_sales
 
 data_path: data/data.csv
 
-parameters:
+parameters: 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)